src/components/structures/MessagePanel.tsx

```diff
@@ -1184,107 +1183,26 @@ class CreationGrouper extends BaseGrouper {
     }
 }
 
-class RedactionGrouper extends BaseGrouper {
+// Wrap consecutive member events in a ListSummary, ignore if redacted.
+// Nowadays, we also include server ACLs, pinned message changes and hidden events in this one grouper,
+// eventually we should rename it. Using multiple groupers leads to a noisier timeline.
+class MemberGrouper extends BaseGrouper {
```

Comment:
While we're here could we please rename this to not be confusing?

Reply:
Yeah true, I didn't think I'd consolidate Redactions at the same time, will do